server/routes/categories.js

```diff
@@ -9,7 +9,7 @@ const {
 } = require('../controllers/categories.js')
 const { authenticate } = require('../middlewares/auth')
 
-router.get('/', authenticate(), getCategories)
+router.get('/', getCategories)
```

Comment:
let's remember to add in the future the middleware authenticate, cause this removing was just with dev purposes